typescript/phoenix-sdk/src/utils/market.ts

```diff
@@ -695,7 +726,22 @@ export function getMarketSwapTransaction({
     },
   });
 
-  return new Transaction().add(ix);
+  swapTransaction.add(swapInstruction);
+
+  // Only close the token accounts if they are native mints
+  if (baseMintKey.equals(NATIVE_MINT)) {
```

Comment:
I think generally people prefer to have normal SOL. The UX of 2 types of SOL is really confusing to the normal user. Jupiter will wrap and unwrap after every swap to avoid the clunky UX

Reply:
That makes a lot of sense